### DISCUSSION TITLE
Is there a way to transfer an existing project owned by one user to another user

### DISCUSSION THREAD

**Discussion by bcstryker** [community/community#45547] [Repositories]
### Select Topic Area

Bug

### Body

I have been developing on a repo I created on my account. To go along with that development I created a project in the repo. In this project there are a bunch of drafts and a few issues. Using it as a personal kanban basically. 

Anyway now I've transferred ownership of the repo to our company github acct. Along with it came all the code and issues. Hooray! Also, when in the repo after transfer, I can see that there is a project. The project I created! When I click it I just get brought to a 404 page. 

On my account, I still see the project and it opens and behaves mostly as I expect. When I click checkmarks or edit existing issues I DO see the changes in the repo's issues on the company account... I'm confused. 

This might be a bug or more dev work might need to be done here. I find it strange that I can have some issues in the project that are issues for a repo owned by a different account, and even interact with them, but then I can't create issues for that repo from that same project. 

Maybe some more thought needs to be put into what happens to a project on a repo when you transfer ownership of the repo? As of now, it seems a number of buggy things occur.

**Comment by CynMali** [comment]
Possibly the more one speaks on it the better the merged ownership outcome.
